etc/ci/performance/README.md

```diff
@@ -0,0 +1,58 @@
+Servo Page Load Time Test
+==============
+
+# Prerequisites
+
+* Python3
+
+# Usage
+## Build Servo
+* Clone the servo repo
+* Compile release build
+* Run `git_log_to_json.sh` in the servo repo, save the output as `revision.json`
+
+## Prepare the local Perfherder
+
+If you want to test the data submission code in `submit_to_perfherder.py`, you can setup a local treeherder VM. If you don't need to test `submit_to_perfherder.py`, you can skip this step.
+
+* Add `192.168.33.10    local.treeherder.mozilla.org` to `/etc/hosts`
+* `git clone https://github.com/mozilla/treeherder; cd treeherder`
+* `vagrant up`
+* `vagrant ssh`
+  * `./bin/run_gunicorn`
+* Outside of vm, open `http://local.treeherder.mozilla.org` and login to create an account
+* `vagrant ssh`
+  * `./manage.py create_credentials slyu slyu@mozilla.com "description"`, the email has to match your logged in user. Remember to log-in through the Web UI once before you run this.
+  * Open a file called `credential.json`. Copy the clinet secrent to your `credential.json`. You can use `credential.json.example` as a template.
+
+
+## Prepare the test runner
+
+* Clone this repo
+* Download [tp5n.zip](http://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip), extract it to `page_load_test/`
+* Put your `servo` binary, `revision.json` and `resources` folder in `servo/`
+* Run `prepare_manifest.sh` to tranform the tp5n manifest to our format
+* `python3 -m virtualenv venv; source venv/bin/activate; pip install treeherder-client`
+* Setup your Treeherder client ID and secret as environment variables `TREEHERDER_CLIENT_ID` and `TREEHERDER_CLIENT_SECRET`
+* Run `test_all.sh`
+* Test results are submitted to https://treeherder.allizom.org/#/jobs?repo=servo
+
+## For Gecko
+
+* Install Firefox Nightly in your PATH
+* Install the `servo-perf-0.0.1.xpi` add-on to your Firefox Nightly
```

Comment:
Can we generate the xpi from the files in the repo instead of including the actual xpi?